README.md

```diff
@@ -3,8 +3,6 @@
 [![Build Status](https://travis-ci.org/lwe/page_title_helper.svg?branch=master)](https://travis-ci.org/lwe/page_title_helper)
 [![Gem Version](https://badge.fury.io/rb/page_title_helper.svg)](http://badge.fury.io/rb/page_title_helper)
 
-This project adheres to [Semantic Versioning](http://semver.org/).
```

Comment:
Why not also keep this in the README, this is one of the first things a user reads and sees and I'd say it's valuable in here as duplicate info too.